LondonBritishLibrary/orient/BLorient12114A.xml

```diff
@@ -0,0 +1,352 @@
+                                <collation>One bifolium after <ref target="#141"/>, one leaf after <ref target="#150"/>, probably two leaves after
+                                        <ref target="#159"/> and two leaves after <ref target="#163"/> are missing.</collation>
+                            </supportDesc>
+                        </objectDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1"> 
+                                    <desc type="OwnershipNote">On the inside of the back cover a label bearing the following note:</desc>
+                                    <q xml:lang="en">Presented by <persName role="owner" ref="PRS14361Smith">Kenneth Smith, 
+                                        <roleName type="title">Esq.</roleName></persName> <date when="1956-07-14">14 July, 1956</date></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#128r"/>
+                                    <desc type="OwnershipNote">Scribbles, including the name of the owner 
+                                        <persName role="owner" ref="PRS14362KidanaWald">Kidāna Wald</persName>.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">The binding is badly damaged.</decoNote>
+                                <decoNote xml:id="b2" type="Boards"><material key="wood"/>Wooden boards.</decoNote>
+                                <decoNote xml:id="b3" type="SlipCase"><material key="leather"></material>Contained in a leather case, also 
+                                    damaged.</decoNote>
+                            </binding>                               
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1800">17th to 18th century</origDate>
+                        </origin> 
+                    </history>
+                    <msPart xml:id="p1">
+                        <msIdentifier>
+                            <idno/>
+                        </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="p1_i1">
+                            <locus from="1r" to="163vb"/>
+                            <title ref="LIT2701Dawit"/>
+                            <msItem xml:id="p1_i1.1">
+                                <locus from="1r" to="128v"/>
+                                <title ref="LIT2000Mazmur"/>
+                                <note>The chapters bear the same title:
+                                    <q xml:lang="gez"><title ref="LIT2000Mazmur">መዝሙር፡ ዘዳዊት።</title></q></note>
```

Comment:
I think he does not mean "chapters"; he means the so-called traditional title of each psalm which exist in several versions. Here, each of them has the title Mazmur za-dawit.